frontend/src/views/camp/Material.vue

```diff
@@ -68,6 +68,8 @@ export default {
         .map((period, idx) => Date.parse(period.end) >= new Date() ? idx : null)
         .filter(idx => idx !== null)
     })
+
+    this.camp().activities().$reload()
```

Comment:
A small improvement to avoid several network requests. 
In the component MaterialTable.vue https://github.com/ecamp/ecamp3/pull/2683/files#diff-850ca229cd293729e94d2201607458788fe4d0e9369f0fa45a605374f75c529dR369 the activities are needed to match `materialItem.materialNode.root` with `activity.rootContentNode`